src/sentry/integrations/github/client.py

```diff
@@ -632,7 +632,11 @@ def get_file(self, repo: Repository, path: str, ref: str, codeowners: bool = Fal
             headers=headers,
         )
 
-        result = contents.content if codeowners else b64decode(contents["content"]).decode("utf-8")
+        result = (
+            contents.content.decode()
```

Comment:
change content from bytes to string in `get_file` since its return type is supposed to be `str`